docs/book/architecture/state_machine.rst

```diff
@@ -121,7 +121,7 @@ would become ``shipping_selected``.
                        args: ["object"]
                    sylius_create_order:
                        on: ["complete"]
-                       do: ["@sm.callback.cascade_transition", "apply"]
+                       do: ["@SM\\Callback\\CascadeTransitionCallback", "apply"]
```

Comment:
If I understand correctly, `sm.callback.cascade_transition` is removed in the `0.4` release of that bundle, which might break the existing Sylius-based application in case one have customised the state machine configuration / written their own configuration for it.

Reply:
Indeed!